simpleclient/src/main/java/io/prometheus/client/CollectorRegistry.java

```diff
@@ -125,31 +125,53 @@ public void clear() {
   public Enumeration<Collector.MetricFamilySamples> metricFamilySamples() {
     return new MetricFamilySamplesEnumeration();
   }
+
+  public Enumeration<Collector.MetricFamilySamples> metricFamilySamples(Set<String> includedNames) {
```

Comment:
This is going to be experimental pending further development of the HTTP API (e.g. will we need regexes?), so should probably get a different name for now.